apollo-router/src/plugins/coprocessor/mod.rs

```diff
@@ -115,6 +115,71 @@ impl Plugin for CoprocessorPlugin<HTTPClientService> {
             builder.wrap_connector(http_connector)
         };
 
+        if matches!(
+            init.config.router.request.context,
+            ContextConf::Deprecated(true)
+        ) {
+            tracing::warn!(
+                "Configuration `coprocessor.router.request.context: true` is deprecated. Use one of the new values, `deprecated` or `all`"
```

Comment:
Is it intentional to not mention the third option `selective` ? Should we provide a link to the official docs ?

Reply:
i was trying to keep it simple, but a docs link might be better (especially as `selective` is the _best_ thing to use...)